x-pack/plugins/security/server/config.ts

```diff
@@ -314,7 +328,31 @@ export function createConfig(
       sortedProviders: Object.freeze(sortedProviders),
       http: config.authc.http,
     },
+    session: getSessionConfig(config.session, providers),
     encryptionKey,
     secureCookies,
   };
 }
+
+function getSessionConfig(session: RawConfigType['session'], providers: ProvidersConfigType) {
+  return {
+    cleanupInterval: session.cleanupInterval,
+    getExpirationTimeouts({ type, name }: AuthenticationProvider) {
```

Comment:
__note:__ alternatively we could set these values to every provider right after config validation, but I wanted to keep that within `session` config "node".